heliopy/data/helper.py

```diff
@@ -145,19 +145,26 @@ def cdf_dict(unit_string):
                         ('#/cm^3', u.cm**-3),
                         ('cm^{-3}', u.cm**-3),
                         ('particles cm^-3', u.cm**-3),
+                        ('n/cc (from moments)', u.cm**-3),
+                        ('n/cc (from fits)', u.cm**-3),
+                        ('km/sec (from fits)', u.km / u.s),
+                        ('km/sec (from moments)', u.km / u.s),
                         ('ionic charge', u.electron),
                         ('earth radii', u.earthRad),
                         ('Re', u.earthRad),
                         ('Degrees', u.deg),
                         ('degrees', u.deg),
+                        ('deg (from fits)', u.deg),
+                        ('deg (from moments)', u.deg),
                         ('#/{cc*(cm/s)^3}', (u.cm**3 * (u.cm / u.s)**3)**-1),
                         ('sec', u.s),
                         ('nT GSE', u.nT),
                         ('nT GSM', u.nT),
                         ('nT DSL', u.nT),
                         ('nT SSL', u.nT),
                         ('msec', u.ms),
-                        ('ionic charge', ionic_charge)])
+                        ('ionic charge', ionic_charge),
+                        ('Spacecraft', False)])
```

Comment:
Why `False` and not `dimensionless_unscaled`?

Reply:
mitplasma function isn't using Spacecraft value in the DataFrame columns so I decided it'd be better if we ignore it.

Reply:
Hmm, it probably should. Could you fix that, and change this to `dimensionless_unscaled`?